src/main/java/com/playmonumenta/libraryofsouls/bestiary/BestiaryArea.java

```diff
@@ -41,17 +39,17 @@ public class BestiaryArea implements BestiaryEntryInterface {
 		NOT_FOUND_ITEM.setItemMeta(meta);
 	}
 
-	public BestiaryArea(BestiaryArea parent, String name, ConfigurationSection config) throws Exception {
+	public BestiaryArea(@Nullable BestiaryArea parent, String name, ConfigurationSection config) throws Exception {
 		mParent = parent;
-		mName = MiniMessage.get().parse(name).decoration(TextDecoration.ITALIC, false);
+		mName = MiniMessage.miniMessage().deserialize(name).decoration(TextDecoration.ITALIC, false);
```

Comment:
I'd consider moving MiniMessage and any other (de)serialization code into Utils.